sdk/core/Azure.Core/api/Azure.Core.net5.0.cs

```diff
@@ -194,10 +194,11 @@ public partial class RequestContext
         public RequestContext() { }
         public System.Threading.CancellationToken CancellationToken { get { throw null; } set { } }
         public Azure.ErrorOptions ErrorOptions { get { throw null; } set { } }
-        public bool HasCustomClassifier { get { throw null; } }
         public void AddPolicy(Azure.Core.Pipeline.HttpPipelinePolicy policy, Azure.Core.HttpPipelinePosition position) { }
         public void ConfigureResponse(int[] statusCodes, bool isError) { }
+        public Azure.Core.ResponseClassifier GetResponseClassifier(System.Func<Azure.RequestContext, Azure.Core.ResponseClassifier> factory, Azure.Core.ResponseClassifier instance) { throw null; }
         public static implicit operator Azure.RequestContext (Azure.ErrorOptions options) { throw null; }
+        public bool TryClassify(int statusCode, out bool isError) { throw null; }
```

Comment:
Make this protected on classifier, so classifier subclasses can call it as part of IsErrorResponse

Reply:
Removed in new approach.